docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM continuumio/miniconda3
+
+MAINTAINER Gilles Bodart <gillesbodart@users.noreply.github.com>
+
+RUN conda create -n env python=3.6
+RUN echo "source activate env" > ~/.bashrc
+ENV PATH /opt/conda/envs/env/bin:$PATH
+
+RUN apt-get -qq -y update
+RUN apt-get -qq -y upgrade
+RUN apt-get -qq -y install \
+        gcc \
+        g++ \
+        wget \
+        curl \
+        git \
+        make \
+        unzip \
+        sudo \
+        vim
+
+# Use C.UTF-8 locale to avoid issues with ASCII encoding
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+# Set the working directory to /app
+WORKDIR /app
+
+COPY ./requirements.txt /app/requirements.txt
+
+# Install any needed packages specified in requirements.txt
+RUN pip install --trusted-host pypi.python.org -r requirements.txt --verbose
+
+
+# Download LASER from FB
+RUN git clone https://github.com/facebookresearch/LASER.git
+
+ENV LASER /app/LASER
+WORKDIR $LASER
+
+RUN bash ./install_models.sh
+
+
+#Installing FAISS
+
+RUN conda install -c pytorch faiss-cpu -y
```

Comment:
This step does not work because it does not install in `env`.
The following solves it: `conda install --name env -c pytorch faiss-cpu -y`